### DISCUSSION TITLE
Not all TailwindCSS colours appear to be available

### DISCUSSION THREAD

**Discussion by pekka** [filamentphp/filament#6559] [Help] [answered]
When I give a column a custom class like described in the [docs](https://filamentphp.com/docs/2.x/tables/columns/getting-started#custom-attributes):

`TextColumn::make('slug')->extraAttributes(['class' => 'bg-gray-200'])`

only a subset of Tailwind's color range appears to be available. 

For example, `bg-teal-500`, `bg-red-100` etc. do not work. 

Is this due to Vite purging unused classes when bundling styles? 

Is there a default way of preventing this?

**Comment by danharrin** [comment, marked as answer]
Using a theme will allow you to build Tailwind yourself with whatever classes you like - https://filamentphp.com/docs/2.x/admin/appearance#building-themes

**Reply by pekka** [reply]
I see. Thank you!

**Reply by pekka** [reply]
This is probably a dumb question, but I haven't been able to figure out how to build Tailwind with all the colour classes using the instructions on the themes page. The `colors` import is already happening in `tailwind.config.js` but that was already there and doesn't appear to do the trick. Would you have another pointer what to look for?

**Reply by raphael-fischer** [reply]
I had the same question, a few minutes ago and the deal is, that you have to integrate EVERY file or path into the content property of tailwind.config.js.
Then every class will be generated from tailwind and its a no-brainer, even for resources, extra pages or javascripts.
I.E. 
```js 
content: [
        './resources/**/*.blade.php',
        './app/Filament/**/*.php', // THAT was the deal maker for myself
        './resources/**/*.js',
        './vendor/filament/**/*.blade.php',
    ],
```
note: the filepath has to be case sensitive!

**Comment by amoktar** [comment]
what about filament 3.x ?
